test_runner/fixtures/types.py

```diff
@@ -0,0 +1,109 @@
+import random
```

Comment:
I wish we could get these types for free by wrapping the rust implementation in pyo3. I might give it a try.

Reply:
Update: I tried this. The easy part was creating python bindings for our rust types. The hard part is importing that python library from poetry as an editable package, so that local changes in the rust code would propagate to what's installed in the mypy env. I wasn't able to do that at all. Not worth pursuing maybe.

Reply:
Yeah, doesn't seem worth the trouble...